tests/file_storage/tests.py

```diff
@@ -123,6 +128,112 @@ def test_file_access_options(self):
         self.storage.delete('storage_test')
         self.assertFalse(self.storage.exists('storage_test'))
 
+    def _test_file_time_getter(self, getter):
+        # We want to check for correct behaviour under both
+        # USE_TZ=True and USE_TZ=False. The tests are
+        # annoyingly similar, since they both set up a
+        # situation where system, Django and UTC are distinct.
+        self._test_file_time_getter_tz_handling_on(getter)
+        self._test_file_time_getter_tz_handling_off(getter)
+
+    @override_settings(USE_TZ=True, TIME_ZONE='Africa/Algiers')
```

Comment:
I used to know what was needed when, but I forgot, so now I'm cargo-culting from tests/timezones/tests.py :-(